src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -151,7 +151,7 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
           fullscreen={fullscreen}
           hasBackButton={hasBackButton}
         >
-          <AnnouncementMessage message={venue.bannerMessage} />
+          <AnnouncementMessage banner={venue.banner} isUser />
```

Comment:
It took me a while to understand what `isUser` means, but I guess it's user-facing announcement. I think we need a better name here, but I can't come up with one. @0xdevalias wdyt